README.md

```diff
@@ -1,32 +1,28 @@
 # JetKit/CDK
 
+[![Tests](https://github.com/jetbridge/jetkit-cdk/actions/workflows/ci.yml/badge.svg)](https://github.com/jetbridge/jetkit-cdk/actions/workflows/ci.yml)
+[![npm version](https://badge.fury.io/js/%40jetkit%2Fcdk.svg)](https://badge.fury.io/js/%40jetkit%2Fcdk)
+
 An anti-framework for building cloud-native serverless applications.
 
-This module provides convenient tools for writing RESTful API views
+This module provides convenient tools for writing Lambda functions, RESTful API views,
 and generating cloud infrastructure with [AWS CDK](https://docs.aws.amazon.com/cdk/latest/guide/home.html).
 
-[![Tests](https://github.com/jetbridge/jetkit-cdk/actions/workflows/ci.yml/badge.svg)](https://github.com/jetbridge/jetkit-cdk/actions/workflows/ci.yml)
-[![npm version](https://badge.fury.io/js/%40jetkit%2Fcdk.svg)](https://badge.fury.io/js/%40jetkit%2Fcdk)
-
 ## Motivation
 
 We want to build maintainable and scalable cloud-first applications, with cloud resources generated from application code.
 
 Using AWS CDK we can automate generating API Gateway routes and Lambda functions from class and function metadata.
+
 Each class or function view is a self-contained Lambda function that only pulls in the dependencies needed for its
 functioning, keeping startup times low and applications modular.
-Get the utility of a minimal web framework without cramming your entire app into a single Lambda.
-
-Optional support for TypeORM using the Aurora Serverless Data API and convenient helpers for CRUD, serialization, tracing, and error handling will be added soon.
 
 ## Documentation
 
-Guides and API reference can be found at [https://jetbridge.github.io/jetkit-cdk/](https://jetbridge.github.io/jetkit-cdk/).
+Guides and API reference can be found at [https://jetkit.dev/docs/](https://jetkit.dev/docs/).
```

Comment:
Totally not relevant to the PR, but you can write bare urls like this `<https://jetkit.dev/docs/>`  to avoid duplicating the same text